tasks/dist/css

```diff
@@ -0,0 +1 @@
+node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
```

Comment:
Running these task files using `npm run *` will run them in the `npm` environment. as you mentioned above @mdo 

https://docs.npmjs.com/misc/scripts#environment